scripts/check_no_api_breakages.sh

```diff
@@ -17,7 +17,7 @@
 ##
 ## This source file is part of the SwiftNIO open source project
 ##
-## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Copyright (c) 2017-2020 Apple Inc. and the SwiftNIO project authors
```

Comment:
```suggestion
## Copyright (c) 2017-2024 Apple Inc. and the SwiftNIO project authors
```

Reply:
This is copy-paste of the header in NIO. The metrics header is above (not in the diff context). I don't update the license header of the source. License headers should be preserved AFAIK.